tests/network/wireguard.pm

```diff
@@ -63,19 +78,53 @@ sub run {
     barrier_wait 'VPN_ESTABLISHED';
     assert_script_run "ping -c10 $vpn_remote";
 
-    zypper_call 'in iperf';
-
-    if (get_var('IS_MM_SERVER')) {
+    if (is_server) {
```

Comment:
Why is `is_server()` better than `get_var('IS_MM_SERVER')`?

Reply:
I though it's nice to read, but we can switch it back if you don't like it

Reply:
In past we used different variables like `IS_SERVER` or `IS_SALT_MASTER` and then it was agreed to switch to `IS_MM_SERVER` so it is everywhere the same. I'd keep it :)